check_commit_message.py

```diff
@@ -0,0 +1,46 @@
+#!/bin/python
```

Comment:
This file is old, check current https://github.com/Tendrl/common/commit/8d821f571594026e8f99c737165da09d899a35e5